client/package.json

```diff
@@ -54,7 +54,8 @@
       "json"
     ],
     "collectCoverageFrom": [
-      "src/**/*.{js,jsx}"
+      "src/**/*.{js,jsx}",
+      "!src/index.js"
```

Comment:
I am not sure if it makes sense to cover this file, but the solutions that I have seen look so ugly that I decided that it will be better just to exclude this file...